nx-X11/programs/Xserver/Xext/xres.c

```diff
@@ -25,7 +25,6 @@
 static int
 ProcXResQueryVersion (ClientPtr client)
 {
-    REQUEST(xXResQueryVersionReq);
```

Comment:
REQUEST is defined as:

    #define REQUEST(type) \
    |       register type *stuff = (type *)client->requestBuffer
`

?! 

"stuff" hardly seems like a good name for something to magically declare in a macro. "register" also seems suspicious. But I guess that is for later.